airflow/config_templates/default_airflow.cfg

```diff
@@ -561,13 +561,22 @@ api_rev = v3
 hide_sensitive_variable_fields = True
 
 [elasticsearch]
+# Elasticsearch host
 elasticsearch_host =
+# Format of the log_id, which is used to query for a given tasks logs
 elasticsearch_log_id_template = {{dag_id}}-{{task_id}}-{{execution_date}}-{{try_number}}
+# Used to mark the end of a log stream for a task
 elasticsearch_end_of_log_mark = end_of_log
 # Qualified URL for an elasticsearch frontend (like Kibana) with a template argument for log_id
 # Code will construct log_id using the log_id template from the argument above.
 # NOTE: The code will prefix the https:// automatically, don't include that here.
 elasticsearch_frontend =
+# Write the task logs to the stdout of the worker, rather than the default files
+elasticsearch_write_stdout = False
```

Comment:
Is this option specific to Elasticsearch remote logging or could it be useful for other loggers too?

Reply:
Also: we don't need to prefix these new settings with `elasticsearch_` as we are already in the `[elasticsearch]` section.

Reply:
@ashb what about `elasticsearch_end_of_log_mark` and another options?

Reply:
> Is this option specific to Elasticsearch remote logging or could it be useful for other loggers too?

I think it's only for ElasticSearch for now, and probably it can be useful for all loggers in future.

Reply:
@ashb agreed about naming, we just kept the convention that was going already. Didn't want to break anyone's configs. Makes sense to drop the prefix though.

Reply:
We could add a deprecation for the old names - it's already supported in Airflow config:

https://github.com/apache/airflow/blob/a8a4d322ee960ef51a03a87db44fe352abb910e6/airflow/configuration.py#L122-L135

Reply:
@andriisoldatenko Do we want to tidy up the config names here or leave it do you think? (all in the same pattern probably makes sense I agree)

Reply:
@ashb depreciation is good point, let me dig into it.